lib/axiom/fs/stdio.js

```diff
@@ -30,8 +30,10 @@ var WritableStream;
  *  ouside world or host (e.g. Ctrl-C).
  * @param {!ReadableStream} ttyin
  * @param {!WritableStream} ttyout
+ * @param {?Element=} overlay
  */
-export var Stdio = function(stdin, stdout, stderr, signal, ttyin, ttyout) {
+export var Stdio = function(stdin, stdout, stderr, signal, ttyin, ttyout,
```

Comment:
Hmm... Stdio is meant to contain streams, not DOM Elements. I understand you need to pass this overlay element around and I don't have a better solution, so I propose this:
1) Don't change the stdio construction
2) Instead, use a named property `overlay` that you set in terminal.js, with a TODO comment explaining we should find a better way to pass state to the underlying import command.
3) Update the `nested_stdio` constructor to copy properties from the parentStdio (using a `for(var key in parentStdio`), so that any additional property is copied (i.e. nothing hard-coded about `overlay`)
4) In the import command, get the overlay element using `cx.stdio['overlay']`
5) In the import command, if the `overlay` property is not set on  stdio, throw an AxiomError.Runtime mentioning overlay is required. The reason is that, when we support remote execution, `overlay` will not be set, and there will be no obvious way to support remote drag & drag.